Source/DafnyLanguageServer/Language/Symbols/ISymbolTableFactory.cs

```diff
@@ -1,22 +1,15 @@
 ﻿using System;
 using System.Threading;
 using Microsoft.Dafny.LanguageServer.Workspace;
+using Compilation = Microsoft.CodeAnalysis.Compilation;
 
 namespace Microsoft.Dafny.LanguageServer.Language.Symbols {
   /// <summary>
   /// Factory definition to generate a symbol table out of a given dafny program and compilation unit.
   /// </summary>
   public interface ISymbolTableFactory {
-    /// <summary>
-    /// Initializes a new symbol table with the given compilation unit.
-    /// </summary>
-    /// <param name="program">The parsed dafny program.</param>
-    /// <param name="compilationUnit">The compilation to create the symbol table of.</param>
-    /// <param name="cancellationToken">A token to cancel the update operation before its completion.</param>
-    /// <returns>A symbol table representing the provided compilation unit.</returns>
-    /// <exception cref="System.OperationCanceledException">Thrown when the cancellation was requested before completion.</exception>
-    /// <exception cref="System.ObjectDisposedException">Thrown if the cancellation token was disposed before the completion.</exception>
```

Comment:
Why delete this comment?

Reply:
Was it useful? I would argue it's not. Even the interface `ISymbolTableFactory` is unused.

Reply:
Given the amount of refactoring going on I just wanted to make sure it wasn't dropped by accident. If you removed it because it wasn't useful that's just fine (and looking closer I agree it seems like "I'm being forced to document every method" rather than "I have something useful to say beyond the signature here" :)